packages/salesforcedx-sobjects-faux-generator/test/fauxgeneratortest.ts

```diff
@@ -26,17 +26,33 @@ describe('generate fields set', function() {
     expect(classText).to.include('String Foo;');
   });
 
-  it('generated faux class should create file with correct fields', async function(): Promise<
+  it('generated faux class should create file with fields that can be in custom SObjects', async function(): Promise<
     void
   > {
-    const field1 = '{"name": "Foo", "type": "string", "referenceTo": []}';
-    const field2 = '{"name": "Foo2", "type" : "boolean", "referenceTo": []}';
-    const sobject1 =
-      '{ "name": "Custom__c", "fields": [ ' +
-      field1 +
-      ',' +
-      field2 +
-      ' ], "childRelationships": [] }';
+    const fieldsHeader = '{ "name": "Custom__c", "fields": [ ';
+    const closeHeader = ' ], "childRelationships": [] }';
+
+    const fields: string[] = [
+      '{"name": "Foo1", "type": "string", "referenceTo": []}',
+      '{"name": "Foo2", "type" : "double", "referenceTo": []}',
+      '{"name": "Foo3", "type" : "boolean", "referenceTo": []}',
+      '{"name": "Foo4", "type" : "currency", "referenceTo": []}',
+      '{"name": "Foo5", "type" : "date", "referenceTo": []}',
+      '{"name": "Foo6", "type" : "datetime", "referenceTo": []}',
+      '{"name": "Foo7", "type" : "email", "referenceTo": []}',
+      '{"name": "Foo8", "type" : "location", "referenceTo": []}',
+      '{"name": "Foo9", "type" : "percent", "referenceTo": []}',
+      '{"name": "Foo10", "type" : "picklist", "referenceTo": []}',
+      '{"name": "Foo11", "type" : "multipicklist", "referenceTo": []}',
+      '{"name": "Foo12", "type" : "textarea", "referenceTo": []}',
+      '{"name": "Foo13", "type" : "encryptedstring", "referenceTo": []}',
+      '{"name": "Foo14", "type" : "url", "referenceTo": []}',
+      '{"name": "Foo15", "type" : "id", "referenceTo": []}'
+    ];
+
+    const fieldsString = fields.join(',');
+    const sobject1 = `${fieldsHeader}${fieldsString}${closeHeader}`;
+
     const sobjectFolder = './';
```

Comment:
Is this safe to use? This seems platform-specific. 
I am a bit surprised that the tests on Appveyor pass. Are you using the `./` in any meaningful way in the tests?

Reply:
Good catch - I copied it from somewhere else and didn't notice..   I'll change it to use process.cwd().